modules/ipi-install-configuring-the-install-config-file.adoc

```diff
@@ -15,6 +15,18 @@ The installation program no longer needs the `clusterOSImage` {op-system} image
 
 . Configure `install-config.yaml`. Change the appropriate variables to match the environment, including `pullSecret` and `sshKey`.
 +
+[NOTE]
```

Comment:
I think that this is too much information to put in a note and that because it's so close to the note immediately before the step that it might get skipped. I think that it would be better to change line 101 and remove this note. Maybe:

> <1> Scale the worker machines based on the number of worker nodes that are part of the {product-title} cluster. Valid options for the `replicas` value are `0` and integers greater than or equal to `2`. Set the number of replicas to `0` to deploy a three-node cluster, which contains only three control plane machines. A three-node cluster is a smaller, more resource-efficient cluster that can be used for testing, development, and production. You cannot install the cluster with only one worker. 

This is also the only reference to a three-node cluster in this topic, so I think it would be more helpful to the user to add more context overall. (References to the three-node cluster are scattered throughout the [equivalent UPI assembly](https://docs.openshift.com/container-platform/4.9/installing/installing_bare_metal/installing-bare-metal.html#installation-three-node-cluster_installing-bare-metal).)